README.md

```diff
@@ -15,7 +15,7 @@ composer require tomasvotruba/class-leak --dev
 Pass directories you want to check:
 
 ```bash
-vendor/bin/class-leak bin src
+vendor/bin/class-leak check bin/ src/
```

Comment:
`bin/` Show that bin is a directory. Shell autocomplete (TAB key) does this also.

Reply:
The name is needed, you're right :+1: 

But I don't like the extra slashes as extra writing.

Reply:
This is too personal for me.
I am so sorry.